dhcpv4/bsdp/vendor_class_identifier_darwin.go

```diff
@@ -0,0 +1,17 @@
+package bsdp
```

Comment:
shouldn't this have the darwin build tag?

Reply:
Nope because it's got the _darwin tag at the end. As long as the other (normal) file has the `// +build !darwin`, it's fine.

Reply:
relying on the other file having the `!darwin` tag seems at risk of inconsistence for future changes. This file would only work on darwin anyway - I think it's reasonable to add the `darwin` build tag anyway (unless there's something wrong in doing that and I'm missing the point)

Reply:
ok, I didn't know that `_darwin` tags the file for the build :D let's merge